deploy/sre-build-test/50-build-test.CronJob.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: sre-build-test
+  namespace: openshift-build-test
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  schedule: "11 */1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: "node-role.kubernetes.io"
+                    operator: In
+                    values:
+                      - infra
+          tolerations:
+          - operator: Equal
+            key: node-role.kubernetes.io
+            value: infra
+            effect: NoSchedule
+          serviceAccountName: sre-build-test
+          restartPolicy: Never
+          containers:
+          - name: sre-build-test
+            image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+            imagePullPolicy: Always
+            command:
+            - /bin/sh
+            - -ec
+            - >-
```

Comment:
this works? very cool.

Reply:
Yes, confirmed the full thing on a stage cluster I had never touched.